cli/cmd/install/install.go

```diff
@@ -420,21 +416,45 @@ func (c *Command) checkForPreviousPVCs() error {
 	return nil
 }
 
-// checkForPreviousSecrets checks for the bootstrap token and returns an error if found.
-func (c *Command) checkForPreviousSecrets() error {
-	secrets, err := c.kubernetes.CoreV1().Secrets("").List(c.Ctx, metav1.ListOptions{LabelSelector: common.CLILabelKey + "=" + common.CLILabelValue})
+// checkForPreviousSecrets checks for Consul secrets that exist in the cluster
+// and returns a message if the secret configuration is ok or an error if
+// the secret configuration could cause a conflict.
+func (c *Command) checkForPreviousSecrets(release release.Release) (string, error) {
+	secrets, err := validation.ListConsulSecrets(context.TODO(), c.kubernetes)
```

Comment:
Yes, it's because I forgot that the parent Command has the `Ctx` field. 🤦🏻 I just looked at the child and said "Oh, no context... I guess I'll just do that later.".